test/js/pageobjects/DocumentSelector.js

```diff
@@ -1,11 +1,9 @@
 import Base from './BasePage.js'
 import { locale, verifyElementCopy } from '../utils/mochaw'
 
-class DocumentSelection extends Base {
-  getTitle() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan'); }
-  getPassport() { return this.$('.onfido-sdk-ui-DocumentSelector-icon-passport')}
+class DocumentSelector extends Base {
   get title() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
-  get subtitle() { return this.$('.onfido-sdk-ui-PageTitle-titleWrapper > div:nth-child(2)')}
+  get subtitle() { return this.$('.onfido-sdk-ui-PageTitle-subTitle')}
```

Comment:
This code is repeated in several screens. Consider creating a class that have all the shared elements. No need to do this as part of this PR.
```get title() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
get subtitle() { return this.$('.onfido-sdk-ui-PageTitle-subTitle')}

copy(lang) { return locale(lang) }
```

Reply:
Good point, thanks. Will try to create a class with common elements to avoid repetition. Probably will do it in separate PR.